### DISCUSSION TITLE
GDL 1.0.0

### DISCUSSION THREAD

**Discussion by alaingdl** [gnudatalanguage/gdl#871] [General]
Happy new year !

Could we consider a 1.0.0 soon ? The sooner the better ! 

I know we all would like to add few codes and improvements, but the 1.0.0rc3 is robust, 
and having a 1.0.0 would be a good point to promote GDL outside the core team.

Alain

**Comment by GillesDuvert** [comment]
Hi,
That would be good, especially since we need to publish more frequently to get GDL updated in the distros.

**Comment by GillesDuvert** [comment]
... And then, the 1.1 shall come soon after, as I'm polishing the new widget support...

**Comment by slayoo** [comment]
@maynardGK @pjb7687 Any objections to make a 1.0 release?

**Comment by pjb7687** [comment]
~I have only one thing: https://github.com/gnudatalanguage/gdl/pull/869~
~This includes GRIB, the last missing package on Windows. No test failures on 64bit. Other than no objections.~

**Reply by pjb7687** [reply]
Merged, no objections now.

**Comment by pjb7687** [comment]
May I ask you going to include windows binaries with the release? If so, I can add another GA script which automatically makes a release with the latest build artifacts.

**Comment by GillesDuvert** [comment]
@slayoo ? What exactly is called a "release"?
I see just [this](https://github.com/gnudatalanguage/gdl/wiki/GDL-on-Windows)

**Comment by slayoo** [comment]
IIUC, what Jeongbin is suggesting (and I'm all for it, but not being a windows user it's hard for me to test/discuss), is that Github offers the option to attach binary files to a release, see e.g.:
https://stackoverflow.com/questions/5207269/how-to-release-a-build-artifact-asset-on-github-with-a-script
"You can also attach binary assets (such as compiled executables, minified scripts, documentation) to a release. "

**Comment by pjb7687** [comment]
Thanks for explaining @slayoo! Just now I released `GDL Workbench` (or `gdlde`) v1.0.0 after days of testing.
This looks like this: https://github.com/gnudatalanguage/gdlde/releases/tag/v1.0.0

**Comment by Gostguest** [comment]
Hi. I have installed GDL, but i cannot find a debugger. Please,help!

**Reply by slayoo** [reply]
Please elaborate. GDL does support STOP, .SKIP, .STEP & .CONTINUE as IDL does (https://www.l3harrisgeospatial.com/docs/_CONTINUE.html), are you referring to any other particular IDL debugging feature? Thanks

**Comment by Gostguest** [comment]
![no debug](https://user-images.githubusercontent.com/78036349/107142236-1536de80-6936-11eb-8826-712805a5bc86.png)

**Reply by GillesDuvert** [reply]
hi @Gostguest , I'm not a user of an IDL/GDL workbench, do you mean you expected a third entry, "debug"?
Note that debugging in GDL is not, by far, as easy as in IDL. This because, probably, GDL was developed to run existing (idl-debugged) code in the first place. A concept that has evolved rapidly in the recent years, so indeed the debugging facilities of GDL should be updated.
